change/@fluentui-web-components-258db20e-84e5-451e-a289-97726ba08b24.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30002&commitId=9f6b5062bd5dfb5d12fa1a3598b6323addbfa63c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Active Appearance - Dark Mode.normal.chromium.png|127| Changed|
|Avatar.Active - Dark Mode.normal.chromium.png|76| Changed|
|Avatar.Active.normal.chromium.png|192| Changed|
|Avatar.Active Appearance.normal.chromium.png|343| Changed|

</details>


  <details><summary><b>RadioGroup</b> 39 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RadioGroup.Default.2nd selected.chromium.png|347| Changed|
|RadioGroup.Default - RTL.2nd selected.chromium.png|342| Changed|
|RadioGroup.Default - Dark Mode.1st selected.chromium.png|280| Changed|
|RadioGroup.Disabled Item.1st selected.chromium.png|226| Changed|
|RadioGroup.Disabled Item - Dark Mode.2nd selected.chromium.png|200| Changed|
|RadioGroup.Default - RTL.1st selected.chromium.png|311| Changed|
|RadioGroup.Default - Dark Mode.normal.chromium.png|280| Changed|
|RadioGroup.Disabled - Dark Mode.2nd selected.chromium.png|265| Changed|
|RadioGroup.Default.normal.chromium.png|316| Changed|
|RadioGroup.Disabled.normal.chromium.png|296| Changed|
|RadioGroup.Disabled Item.normal.chromium.png|226| Changed|
|RadioGroup.Disabled - Dark Mode.normal.chromium.png|262| Changed|
|RadioGroup.Disabled.1st selected.chromium.png|298| Changed|
|RadioGroup.Default.1st selected.chromium.png|316| Changed|
|RadioGroup.Disabled Item.2nd selected.chromium.png|226| Changed|
|RadioGroup.Default Checked.normal.chromium.png|341| Changed|
|RadioGroup.Default Checked.2nd selected.chromium.png|347| Changed|
|RadioGroup.Horizontal Stacked - RTL.normal.chromium.png|7164| Changed|
|RadioGroup.Horizontal Stacked - RTL.2nd selected.chromium.png|7273| Changed|
|RadioGroup.Disabled - Dark Mode.1st selected.chromium.png|265| Changed|
|RadioGroup.Horizontal.1st selected.chromium.png|316| Changed|
|RadioGroup.Default - RTL.normal.chromium.png|311| Changed|
|RadioGroup.Disabled Item - Dark Mode.normal.chromium.png|200| Changed|
|RadioGroup.Default Checked.1st selected.chromium.png|316| Changed|
|RadioGroup.Vertical - RTL.1st selected.chromium.png|18749| Changed|
|RadioGroup.Vertical.1st selected.chromium.png|18748| Changed|
|RadioGroup.Horizontal.normal.chromium.png|316| Changed|
|RadioGroup.Horizontal Stacked.1st selected.chromium.png|7240| Changed|
|RadioGroup.Horizontal Stacked.normal.chromium.png|7165| Changed|
|RadioGroup.Horizontal.2nd selected.chromium.png|347| Changed|
|RadioGroup.Default - Dark Mode.2nd selected.chromium.png|287| Changed|
|RadioGroup.Vertical - RTL.2nd selected.chromium.png|18749| Changed|
|RadioGroup.Vertical - RTL.normal.chromium.png|18749| Changed|
|RadioGroup.Disabled Item - Dark Mode.1st selected.chromium.png|200| Changed|
|RadioGroup.Horizontal Stacked.2nd selected.chromium.png|7274| Changed|
|RadioGroup.Horizontal Stacked - RTL.1st selected.chromium.png|7238| Changed|
|RadioGroup.Disabled.2nd selected.chromium.png|298| Changed|
|RadioGroup.Vertical.normal.chromium.png|18748| Changed|
|RadioGroup.Vertical.2nd selected.chromium.png|18748| Changed|

</details>


  <details><summary><b>Switch</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch.Checked - RTL.normal.chromium.png|97| Changed|
|Switch.Checked - RTL.hover.chromium.png|104| Changed|

</details>


  <details><summary><b>TextInput</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextInput.Appearance Filled Darker - Dark Mode.normal.chromium.png|8164| Changed|
|TextInput.Appearance Filled Darker.normal.chromium.png|7297| Changed|
|TextInput.Appearance Underline - Dark Mode.normal.chromium.png|125| Changed|
|TextInput.Disabled.normal.chromium.png|199| Changed|
|TextInput.Appearance Underline.normal.chromium.png|91| Changed|
|TextInput.Appearance Filled Lighter - Dark Mode.normal.chromium.png|7791| Changed|
|TextInput.Disabled - Dark Mode.normal.chromium.png|203| Changed|
|TextInput.Appearance Filled Lighter.normal.chromium.png|7669| Changed|
|TextInput.Size Small.normal.chromium.png|2537| Changed|
|TextInput.Size Large.normal.chromium.png|2564| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

Reply:
This looks okay to me but I let a WC Codeowner mark this as resolved after reviewing.